IPython/terminal/ipapp.py

```diff
@@ -154,6 +154,7 @@ def make_report(self,traceback):
 
 aliases = dict(base_aliases)
 aliases.update(shell_aliases)
+aliases['term-title-format'] = 'TerminalInteractiveShell.term_title_format'
```

Comment:
It does not show up, but it does work 
<img width="694" alt="screen shot 2017-05-11 at 13 36 05" src="https://cloud.githubusercontent.com/assets/335567/25970825/d3e90264-364e-11e7-8b2b-afad10b3b3d1.png">

So that's a separate bug.